src/main/java/org/folio/circulation/domain/Item.java

```diff
@@ -423,4 +423,17 @@ public Item withLoanType(JsonObject newLoanTypeRepresentation) {
       this.primaryServicePoint,
       newLoanTypeRepresentation);
   }
+
+  public Item withLastCheckIn(JsonObject lastCheckInRepresentation) {
```

Comment:
If the `UpdateItem` class does the mapping to writing the last check in information based upon specific parameters, what is this method used for?